app/views/admin/edition_workflow/_unwithdraw_modal.html.erb

```diff
@@ -9,7 +9,7 @@
           <h3 class="modal-title">Are you sure you want to unwithdraw this?</h3>
         </div>
         <div class="modal-body">
-          <p>This will create a new edition and immediately publish it.</p>
+          <p>This will create a new edition and immediately publish it silently. This will not trigger email alerts.</p>
```

Comment:
I think you can probably lose the 'silently'. It's a less clear way of saying 'this will not trigger email alerts'.